### PR TITLE
Magento_Cms: avoid using deprecated escape* methods from AbstractBlock

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Static Tests, Integration Tests

File: app/code/Magento/Cms/view/adminhtml/templates/url_filter_applier.phtml [hunk 1]
```diff
@@ -4,8 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Backend\Block\Template */
-/** @var \Magento\Framework\Escaper $escaper */
+/**
+ * @var $block \Magento\Backend\Block\Template
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Cms/view/adminhtml/templates/url_filter_applier.phtml]:
```suggestion
 * @var \Magento\Backend\Block\Template $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml [hunk 2]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader */
+/**
+ * @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml]:
```suggestion
 * @var \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml [hunk 3]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Files */
+/**
+ * @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Files
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml]:
```suggestion
 * @var \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Files $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content.phtml [hunk 4]
```diff
@@ -4,10 +4,13 @@
  * See COPYING.txt for license details.
  */
 
- /** @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content */
+ /**
+  * @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content
+  * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Cms/view/adminhtml/templates/browser/content.phtml]:
```suggestion
  * @var \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content $block
  * @var \Magento\Framework\Escaper $escaper
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content.phtml]: fixed

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml]: fixed

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml]: fixed

File: app/code/Magento/Cms/view/adminhtml/templates/url_filter_applier.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cms/view/adminhtml/templates/url_filter_applier.phtml]: fixed

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml [hunk 5]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader */
+/**
+ * @var \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml]:
```suggestion
/**
 * @var \Magento\Cms\Block\Adminhtml\Wysiwyg\Images\Content\Uploader $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content/uploader.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Pull request closed without merging**